passes/memory/memory_bram.cc

```diff
@@ -1073,6 +1107,43 @@ void handle_cell(Cell *cell, const rules_t &rules)
 						i+1, log_id(bram.name), bram.variant);
 				goto next_match_rule;
 			}
+			
+			for (auto synattr: match.attr_syn) {
+				auto it = cell->attributes.find(synattr.first);
+				if(it != cell->attributes.end()) {
+					if(it->first == "\\logic_block" || it->first == "\\ram_style") {
```

Comment:
Should we still hard code these? This seems a backward step from what we had before.